Proofs/Experiments/MemoryAliasing.lean

```diff
@@ -23,3 +23,24 @@ theorem mem_automation_test
 
 /-- info: 'mem_automation_test' depends on axioms: [propext, Classical.choice, Quot.sound] -/
 #guard_msgs in #print axioms mem_automation_test
+
+theorem mem_automation_test_2
+  (h_n0 : n0 ≠ 0)
+  (h_no_wrap_src_region : mem_legal' src_addr ((n0 <<< 4) - 1))
+  (h_no_wrap_dest_region : mem_legal' dest_addr ((n0 <<< 4) - 1))
+  (h_s0_src_dest_separate :
+    mem_separate' src_addr  ((n0 <<< 4) - 1)
+                  dest_addr ((n0 <<< 4) - 1)) :
```

Comment:
```suggestion
    mem_separate' src_addr  (n0 <<< 4)
                  dest_addr (n0 <<< 4) :
```

Reply:
Looks like the `- 1` got left over from the non-primed defns where we need to compute the last address.